routes/start.py

```diff
@@ -1,11 +1,14 @@
 from aiohttp import web
+from json import loads
 
 from server import GameServer
 
 async def start(request: web.Request):
-    login = await request.text()
+    body = loads(await request.text())
```

Comment:
In the opening handshake, the client would send their login and receive a unique token. In subsequent requests, there would then be no need to send the login again, only the token is required.